frontend/src/store/index.js

```diff
@@ -25,6 +28,7 @@ export default createStore(
         },
       }),
       logger,
+      thunk,
```

Comment:
Do we need thunk?

Reply:
Thunk allows our actions to return functions. I use this quite a lot when composing actions together: https://github.com/payload-app/payload/pull/5/files#diff-48bb2e3e3e5708d8083b1cff1fff3949R40

Reply:
Ah I see the pattern! What if you dispatched one action `@@init` or `APP_INIT` and caught the init action in middleware which could trigger those actions closer to the code? Then no need for thunk! 

Not that we shouldn't use it, I think that `thunk` adds complexity!

Reply:
That makes sense for doing app init stuff, but I don't think it makes sense to do all action composition in different middlewares. 

My usage here isn't best practice. There should be something like a `fetchRepos` action that then dispatches both of those actions, it shouldn't be in the connect function there. That was just me being lazy.

Maybe we should chat about this at some point, might be a faster convo 😄